irr/src/OpenGLES2/Driver.cpp

```diff
@@ -98,8 +96,6 @@ void COpenGLES2Driver::initFeatures()
 
 		if (FeatureAvailable[IRR_GL_OES_depth_texture]) {
 			TextureFormats[ECF_D16] = {GL_DEPTH_COMPONENT, GL_DEPTH_COMPONENT, GL_UNSIGNED_SHORT};
-			if (FeatureAvailable[IRR_GL_OES_depth32])
-				TextureFormats[ECF_D32] = {GL_DEPTH_COMPONENT, GL_DEPTH_COMPONENT, GL_UNSIGNED_INT};
```

Comment:
If I'm reading https://registry.khronos.org/OpenGL/extensions/OES/OES_depth_texture.txt correctly it already defines the combination of `(GL_DEPTH_COMPONENT, GL_UNSIGNED_INT)`

Reply:
Wow, I see, this is more messy than I thought...